include/tvm/tir/schedule/schedule.h

```diff
@@ -405,6 +405,36 @@ class ScheduleNode : public runtime::Object {
   virtual BlockRV CacheWrite(const BlockRV& block_rv, int write_buffer_index,
                              const String& storage_scope,
                              const Array<BlockRV> consumer_blocks = {}) = 0;
+  /*!
+   * \brief Create a block that reads a buffer region into a read cache. It requires:
+   * 1) There is at most one block who writes the buffer in the scope.
+   * 2) The scope block have stage-pipeline property.
+   * Compared to cache read, the indices to access allocated cache buffer is customized by user.
+   * \param block_rv The consumer block of the target buffer.
+   * \param read_buffer_index The index of the buffer in block's read region.
+   * \param storage_scope The target storage scope.
+   * \param index_map User defined indices to access allocated cache buffer, maps from block iter
+   * vars. \param consumer_blocks An optional list of consumers to read from cache directly. \return
+   * The cache stage block.
+   */
```

Comment:
```suggestion
   * \param read_buffer_index The index of the buffer in block's read region.
   * \param storage_scope The target storage scope.
   * \param index_map User defined indices to access allocated cache buffer, maps from block iter
   * vars. 
   * \param consumer_blocks An optional list of consumers to read from cache directly. 
   * \return The cache stage block.
   */
```

Reply:
Hi, my original code looks like what you suggested but it doesn't pass CI, see [here](https://ci.tlcpack.ai/blue/organizations/jenkins/tvm-lint/detail/PR-14161/4/pipeline), and by default, clang-format would format the code like this:
```
* \param index_map User defined indices to access allocated cache buffer, maps from block iter
* vars. \param consumer_blocks An optional list of consumers to read from cache directly. \return
```
do you have any suggestions on how to change the default behavior of clang-format and CI?